PayButton/paybutton-server
# Feat/411 6 1 subscribe transactions

Description
---
Subscribes to receive broadcast of new transactions in the blockchain (grpc client only). They should be added to our database at the same time as they happen in the blockchain, first as an unconfirmed transaction, then as a confirmed transaction when it confirms.

Test plan
---

- add an address you control to the application (one that will receive the transfer)
- run: yarn docker c, then flushall, then exit
- run: yarn docker jr
- run: yarn docker jw
- transfer to the address in step 1
- run database select * from paybutton.Transaction, you will see your transaction added as unconfirmed fist, then as confirmed
